run.sh

```diff
@@ -51,6 +51,7 @@ leaderVoteAccountPubkey=$(\
 
 solana-genesis \
   --lamports 1000000000 \
+  --lamports-per-signature 1 \
```

Comment:
this is gonna break voting...

Reply:
The `./run.sh` cluster only ever contains a single node so I don't think it will here.  I want to turn it on here so that I force solana-web3.js to start having to  deal with fees

But yes more generally it totally will, why is why I didn't turn on fees for `net/`-based testnets just yet.   Are you *fixing* voting or want me to?

Reply:
Oh no you're right.  Once I fixed a different bug `./run.sh` started to fail after the 42nd vote.

Reply:
I am not fixing vote right now, but would like to next.